src/persistence/db/contact.rs

```diff
@@ -0,0 +1,309 @@
+use std::collections::HashMap;
+
+use super::Result;
+use async_trait::async_trait;
+use serde::{Deserialize, Serialize};
+use surrealdb::{engine::any::Any, Surreal};
+
+use crate::{persistence::ContactStoreApi, service::contact_service::IdentityPublicData};
+
+#[derive(Clone)]
+pub struct SurrealContactStore {
+    db: Surreal<Any>,
+    table: String,
```

Comment:
Would it be simpler, if this would just be a const `&'static str`? Or do we potentially have to change this at runtime?